### DISCUSSION TITLE
Upgrade react-native-firebase from v6 to v8

### DISCUSSION THREAD

**Discussion by omrip2** [invertase/react-native-firebase#4410] [Q&A] [answered]
Hello,

Currently, I'm using:
    "@react-native-firebase/analytics": "^6.3.4",
    "@react-native-firebase/app": "^6.3.4",
    "@react-native-firebase/auth": "^6.3.4",
    "@react-native-firebase/crashlytics": "^6.3.4"

I wanted to ask how can I upgrade my libraries to the latest version.
I need to update in order to use the Crashlytics SDK which is not based on Fabric.
Moreover, does this upgrade should effects my js code.

Thank you very much for your help.

**Comment by mikehardy** [comment, marked as answer]
Hi there - when upgrading, go to the releases page https://rnfirebase.io/releases/ and scan the changes between the versions you are on and the current versions. We follow semver so unless there is a major version change there is nothing to do, and when the major version changes happen I did my best to include in the notes right on each release exactly what you need to do

**Reply by andersonaddo** [reply]
I'd also like to add that you're going to need to change a couple native things when upgrading your crashlytics. Re-read those docs and search for past github issues if you face any trouble - it's a solved problem.
